source/_components/blink.markdown

```diff
@@ -33,15 +33,16 @@ Once loaded, your front end will have the following components:
 * A sensor per camera that reports battery level.
 * A sensor per camera that reports unread notification (ie. detected motion events).
 
-Since the cameras are battery operated, the images are only updated in Home Assistant when the user manually forces a new photo.  The image can only be updated in Home Assistant every 60 seconds in order to not overwhelm Blink's servers with API requests.
+Since the cameras are battery operated, the images are only updated in Home Assistant when the user manually forces a new photo.  This image can be updated with the ``blink.snap_picture`` service followed by a ``blink.force_update`` service call to force Home Assistant to request an update from Blink's servers.  If the ``blink.force_update`` service is not called, the image will be updated within a 180 second interval, set so that automatic server requests don't overwhelm the Blink API.  Any camera-specific sensor typically only has its value updated when the image is updated, so relying on, for example, the temperature sensor reporting accurate data is not recommended.
```

Comment:
Double spaces between sentences.
Unsure what your trying to say in the last sentence.